src/components/NewService.vue

```diff
@@ -312,6 +315,13 @@
       .catch(error => { console.log(error) }))
       // Once Services, Clusers and Service Types obtained, generate config template
       axios.all(promises).then(response => {
+        this.service_types = this.main_catalog.concat(this.custom_catalog)
+        // Check if at leats one service type to deploy exists
+        if (this.service_types.length > 0) {
+          this.service_type_id = this.service_types[0].id
+        } else {
+          this.cannotRegister = true
```

Comment:
When is this flag set to false?

Reply:
@taliaga It's false by default, but if there are no services to deploy it's set to true, since there's nothing to deploy. It's also used to disable the register buttons when a user picks a service type that requries some dependant service (e.g. IoT Agent needs Mongo), but no services of that type are deployed, therefore you can't deploy the selected service type.

Reply:
@taliaga It's set to false when the page is loaded, and when the user changes the service type being deployed from the dropdown menu, since the form is regenerated then